backend/src/mockExamQuestion/mockExamQuestion.controller.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  ParseIntPipe,
+} from '@nestjs/common';
+import { ApiResponse } from '@nestjs/swagger';
+import { MockExamQuestionsGenerateResponseDto, UserRole } from '@osk/shared';
+import { Roles } from '../common/guards/roles.decorator';
+import { assertNever } from '../utils/assertNever';
+import { MockExamQuestionService } from './mockExamQuestion.service';
+
+@Controller('questions')
+@Roles(UserRole.Trainee)
+export class MockExamQuestionsController {
+  constructor(
+    private readonly mockExamQuestionsService: MockExamQuestionService,
+  ) {}
+
+  @ApiResponse({
+    type: MockExamQuestionsGenerateResponseDto,
+  })
```

Comment:
Using `description` field let's add some description on what this endpoint does as it's not super obvious at first sight